src/decode.h

```diff
@@ -288,6 +288,12 @@ typedef struct rv_insn {
 #if RV32_HAS(EXT_C)
     uint8_t shamt;
 #endif
+
+#if RV32_HAS(EXT_F)
+    /* rounding mode */
```

Comment:
Outline the various rounding modes in this comment as specified in the RISC-V standard.